File: C/Ex13.c
#include <stdio.h>
#include <stdlib.h>

int main(int argc, char *argv[]){
    float b, h, a;

    printf("Calculo da area de um retangulo\n");

    printf("Digite a base: ");
    scanf("%f", &b);
    printf("Digite a altura: ");
    scanf("%f", &h);

    a= b*h;
    printf("Sua area eh = %.2f\n", a);
    if (a<100){
    printf("Terreno pequeno");
            }
    
    else if (a>100) {
        printf("Terreno grande");
            }

    return 0;
    }

File: Python/Ex13.py
print("Calculo da area de um retangulo")
b= float(input("Digite a base: "))
h= float(input("Digite a altura: "))
a=b*h
if (a<100):
    print(f"Sua area eh= {a:.2f}")
    print("Terreno pequeno!")
elif (a>100):
    print(f"Sua area eh= {a:.2f}")
    print("Terreno grande!")